modules/interactive_music/audio_stream_interactive.cpp

```diff
@@ -486,7 +486,7 @@ void AudioStreamInteractive::_bind_methods() {
 
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "initial_clip", PROPERTY_HINT_ENUM, "", PROPERTY_USAGE_DEFAULT), "set_initial_clip", "get_initial_clip");
 
-	ADD_PROPERTY(PropertyInfo(Variant::INT, "clip_count", PROPERTY_HINT_RANGE, "1," + itos(MAX_CLIPS), PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_ARRAY, "Clips,clip_,page_size=999,unfoldable,numbered,swap_method=_inspector_array_swap_clip,add_button_text=" + String(RTR("Add Clip"))), "set_clip_count", "get_clip_count");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "clip_count", PROPERTY_HINT_RANGE, "1," + itos(MAX_CLIPS), PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_ARRAY, "Clips,clip_,page_size=999,unfoldable,numbered,swap_method=_inspector_array_swap_clip,add_button_text=" + ETR("Add Clip")), "set_clip_count", "get_clip_count");
```

Comment:
`ETR` is not the correct macro here, but according to [the extraction script](https://github.com/godotengine/godot-editor-l10n/blob/6151b80bf224d53731a053a58fb94ab32e241e16/scripts/extract_editor.py#L50-L60), there is no macro which marks a string as "translatable" without actually doing the translation at runtime as well.

Am I missing something or is this something new that would have to be added?